src/main/scala/cromwell/backend/provider/local/LocalBackend.scala

```diff
@@ -360,13 +361,16 @@ class LocalBackend(task: TaskDescriptor) extends BackendActor with StrictLogging
     * @return TaskStatus with final status of the task.
     */
   private def processOutputResult(processReturnCode: Int, outputsExpressions: Seq[(String, OutputStmtEval)]): TaskFinalStatus = {
-    if (outputsExpressions.filter(_._2.rhs.isFailure).size > 0) {
+    if (outputsExpressions.exists(_._2.rhs.isFailure)) {
       TaskFinalStatus(Status.Failed, FailureTaskResult(new IllegalStateException("Failed to evaluate output expressions.",
         outputsExpressions.filter(_._2.rhs.isFailure).head._2.rhs.failed.get), processReturnCode, stderr.toString))
     } else {
       try {
+        // FIXME this code is not properly structured to deal with asynchronous hash computation.  The Await.result is
+        // gross but safe since the code isn't actually pulling down Docker image hashes, so the `Future` we're getting
+        // back is currently just a `Future.successful`.
         TaskFinalStatus(Status.Succeeded, SuccessfulTaskResult(outputsExpressions.map(
-          output => output._1 -> resolveOutputValue(output._2)).toMap, new ExecutionHash(computeHash, None)))
+          output => output._1 -> resolveOutputValue(output._2)).toMap, Await.result(computeHash, Duration.Inf)))
```

Comment:
What happens if computeHash fails? It will continue being a SuccesfulTaskResult?